README.md

```diff
@@ -95,9 +49,53 @@ python app.py
 ngrok http 3000
 ```
 
-## AsyncApp Setup
+## Listening for events
+Apps typically react to a collection of incoming events, which can correspond [Events API events](https://api.slack.com/events-api), [actions](https://api.slack.com/interactivity/components), [shortcuts](https://api.slack.com/interactivity/shortcuts), [slash commands](https://api.slack.com/interactivity/slash-commands) or [options requests](https://api.slack.com/reference/block-kit/block-elements#external_select). For each type of
+request, there's a method to build a listener function.
+
+```python
+// Listen for an event from the Events API
```

Comment:
how about mentioning decorator first and then having a list of `app.xxx`?

The recommended way is to use Python decorators.

```python
@app.event(event_type)
def handle_event(event):
    pass
```

Here is a list of available methods to register listeners:
```python
app.event(event_type, matcher)
app.message([pattern ,] fn)
```

Please remove `@` from the examples in the list.